### ISSUE TITLE
jar with dependencies

### ISSUE THREAD

**Issue by santhoshTpixler** [authorjapps/zerocode#192]
It would be nice to have an additional jar in the maven central `zerocode-tdd-jar-with-dependencies` , which will help people using zerocode without dependency management tools. 

It should be an uber jar with all the necessary dependencies packaged into it. 

The example screen shot of others doing it. 
![image](https://user-images.githubusercontent.com/10362257/51421686-e111e900-1bc7-11e9-974a-762c976a8261.png)

**Comment by authorjapps** [comment]
Yes, good one. let's generate it via the `shade` plugin and publish it.
e.g.
```xml
<plugin>
                <groupId>org.apache.maven.plugins</groupId>
                <artifactId>maven-shade-plugin</artifactId>
                <version>3.1.0</version>
                <executions>
                    <execution>
                        <phase/>
                    </execution>
                </executions>
</plugin>
```

>Also we might need a ticket to provide a command line tool or `mvn test` examples for picking n executing a test directly from resource
Will address this in a separate ticket.

**Comment by authorjapps** [comment]
Latest 1.3.2 fat-jar aka uber-jar can be downloaded from here.
Link:
https://github.com/authorjapps/zerocode/releases/download/zerocode-tdd-parent-1.3.2/zerocode-tdd-1.3.3-SNAPSHOT.jar

Generally going forward, it can be downloaded from the `assets` section.
![release](https://user-images.githubusercontent.com/12598420/52523279-a4e21d80-2c87-11e9-8920-d8b309841c66.png)

**Comment by santhoshTpixler** [comment]
Uber jar missing in latest release 
![image](https://user-images.githubusercontent.com/10362257/54031580-2b314700-41d5-11e9-9142-6d84ad8577d7.png)

**Comment by authorjapps** [comment]
yes, will sort that today 👍 . Somehow we need to automate this as uploading takes around 5 to 10min.

**Comment by authorjapps** [comment]
File support available in `1.3.17` release.
